test/d-b-n_smoke/arrays.html

```diff
@@ -162,6 +162,14 @@
                     done();
                 }, 100);
             });
+
+            /** This should be used with spy on patchsent event */
+            function expectPatchSent(operation) {
```

Comment:
I think `expect(operationObject).to.be.sent` has a benefit of being chainable, with other chai.

I would say that the fact that those two are related and tightly related is extremely unreadable:

```js
ppclient.addEventListener('patchsent', patchSentEventCallback);
// ...
expectPatchSent(operationObject('add', '/people_push/1', {first: 'Abraham'}));
```

I would suggest something like

```js
expect(patchSentEventCallback).to.have.ben.calledWithJSONPatch(operation)
```
which will be a sugar on top of regular `calledWith` profiled for (de-versioned) JSON Patch